app/services/order_purchaser.rb

```diff
@@ -62,9 +62,8 @@ def order_in_the_past!
     # update order detail statuses if you've changed it while acting as
     if acting_as? && order_status_id.present?
       order.backdate_order_details!(user, order_status_id)
```

Comment:
It seems like the only place `complete_past_reservations!` and `backdate_order_details!` are used is here. Just curious, what would that look like if we pulled those methods out into `OrderPurchaser`? Would that add any clarity?